actions/lib/purge_remote_folder.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/python
+
+import argparse
+import datetime
+import json
+import logging
+import os
+import shutil
+import socket
+
+
+class PurgeRemoteFolders(object):
+
+    def __init__(self, biotank_archive_folder, age_in_days, dryrun):
+        self.biotank_archive_folder = str(biotank_archive_folder)
```

Comment:
Since this relates to removing data, I think that perhaps we could do a bit more defensive programming here. E.g. checking that the `biotank_archive_folder` is not empty, and perhaps that the `age_in_days` is not zero, or at least not negative?

Reply:
good idea